.github/workflows/frontend-ci.yml

```diff
@@ -58,36 +58,16 @@ jobs:
       - run: cp .env.template .env
       - run: ./scripts/extract-supabase-anon-key.sh
       - run: pnpm test:turbo
-      - name: Check for diff in generated types
+      - run: pnpm --filter @liam-hq/db supabase:gen
+      - name: Check for diff in generated types and schema.sql
+        env:
+          # check two files.
+          GENERATED_FILES: "frontend/packages/db/supabase/database.types.ts frontend/packages/db/schema/schema.sql"
         run: |
-          pnpm --filter @liam-hq/db supabase:gen
-          git add frontend/packages/db/supabase/database.types.ts
-          if ! git diff HEAD --ignore-space-at-eol --exit-code frontend/packages/db/supabase/database.types.ts; then
-            echo "Generated types differ from committed file."
-            git diff HEAD frontend/packages/db/supabase/database.types.ts
+          if ! git diff HEAD --ignore-space-at-eol --exit-code ${{ env.GENERATED_FILES }}; then
+            echo "Generated types and schema.sql differ from committed files."
             exit 1
           else
-            echo "Generated types are up-to-date."
-            exit 0
-          fi
-      - uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
-        id: migrations-changes
-        with:
-          filters: |
-            has-changes:
-              - '.github/workflows/frontend-ci.yml'
-              - 'frontend/packages/db/schema'
-              - 'frontend/packages/db/supabase/migrations'
```

Comment:
Since `supabase:gen`  is fast (around 2 seconds), I realized that filtering is unnecessary.


<img width="1012" alt="スクリーンショット 2025-03-28 17 45 19" src="https://github.com/user-attachments/assets/8e35c32b-dc16-43f9-ad45-2a5ba8455477" />